firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1273,6 +1273,11 @@ extension String {
 extension String {
     public struct Microsurvey {
         public struct Prompt {
+            public static let LogoImageA11yLabel = MZLocalizedString(
+                key: "Microsurvey.Prompt.LogoImage.AccessibilityLabel.v129",
```

Comment:
@thatswinnie I tag this v129 to avoid confusion with the string freeze, but should I wait until v128 nightly is created or does it not matter. 

cc: @adudenamedruby

Reply:
we should be adding strings versions with the tag of the version in which the string will be translated. THat will allow us to more easily change the `key` value as we move forward, if a change is needed.

Reply:
okay, so `v128` is okay as long as we're okay with it not being translated in time. will update to `v128`!